pkg/jobs/helpers_test.go

```diff
@@ -1,148 +0,0 @@
-// Copyright 2017 The Cockroach Authors.
```

Comment:
Minor, but I think changing this from `helpers_test.go` to live in test_helpers.go` isn't just a cosmetic change.  IIRC, go excludes _test.go files from the non-test binary, meaning that the exported functions and such aren't compiled into the executable.

Reply:
i actually did this intentionally because I wanted to be able to use FakeResumer in other test files outside of `pkg/jobs`. Let me know if I'm missing what you're suggesting here.

Reply:
there are a bunch of vars and constants, and some methods on jobs.Job, that are exported only in this file.

Maybe just move FakeResumer to a jobstest/resumer.go package instead?

Reply:
Ah TIL about jobstest, that sounds like the right place for it. I'll rename this to `helpers_test.go` after doing that.

Reply:
all done, check it out

Reply:
actually...dep cycle because we have a test file in `pkg/jobs` that imports `jobstest`. And `jobtest` pulls in `jobs` to get a handle to `OnPauseRequestFunc`. hm

Reply:
im going to just not use the type definition in the resumer.go file to break this cycle